### ISSUE TITLE
How navigate and extract information in results.pandas().xyxy[0]

### ISSUE THREAD

**Issue by ghost** [ultralytics/yolov5#7180]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello !, i want to extract information like the coordinates of the differents labels for putting them in others variables.

results.pandas().xyxy[0]
#      xmin             ymin    xmax   ymax  confidence  class    name
# 0  749.50           43.50   1148.0  704.5    0.874023      0  person
# 1  433.50          433.50   517.5  714.5    0.687988     27     tie
# 2  114.75          195.75  1095.0  708.0    0.624512      0  person
# 3  986.00           304.00  1028.0  420.0    0.286865     27     tie

So for exemple if i want to extract the 43.50 so ((x = 1, y = 0) i suppose) what is the right line to type.

Thank anyone for the help and the time investment.

**Comment by ghost** [comment]
And i sorry for the indentation absolutly disgusting ^^

**Comment by glenn-jocher** [comment]
@Lanzull 👋 Hello! Thanks for asking about **handling inference results**. YOLOv5 🚀 [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading) models allow for simple model loading and inference in a pure python environment without using `detect.py`. 

### Simple Inference Example

This example loads a pretrained YOLOv5s model from PyTorch Hub as `model` and passes an image for inference. `'yolov5s'` is the YOLOv5 'small' model. For details on all available models please see the [README](https://github.com/ultralytics/yolov5#pretrained-checkpoints). Custom models can also be loaded, including custom trained PyTorch models and their [exported](https://docs.ultralytics.com/yolov5/tutorials/model_export) variants, i.e. ONNX, TensorRT, TensorFlow, OpenVINO YOLOv5 models.

```python
import torch

# Model
model = torch.hub.load('ultralytics/yolov5', 'yolov5s')  # or yolov5m, yolov5l, yolov5x, etc.
# model = torch.hub.load('ultralytics/yolov5', 'custom', 'path/to/best.pt')  # custom trained model

# Images
im = 'https://ultralytics.com/images/zidane.jpg'  # or file, Path, URL, PIL, OpenCV, numpy, list

# Inference
results = model(im)

# Results
results.print()  # or .show(), .save(), .crop(), .pandas(), etc.

results.xyxy[0]  # im predictions (tensor)
results.pandas().xyxy[0]  # im predictions (pandas)
#      xmin    ymin    xmax   ymax  confidence  class    name
# 0  749.50   43.50  1148.0  704.5    0.874023      0  person
# 2  114.75  195.75  1095.0  708.0    0.624512      0  person
# 3  986.00  304.00  1028.0  420.0    0.286865     27     tie
```
<img width="550" src="https://user-images.githubusercontent.com/26833433/149679662-b2021c78-59b2-4f52-b768-1e5161537c85.png">

See [YOLOv5 PyTorch Hub Tutorial](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading) for details.

Good luck 🍀 and let us know if you have any other questions!

**Comment by mm0806son** [comment]
Hi,

I fell into this thread while searching for a solution for the same. Although this question is marked completed, I think the code in tutorial as mentioned before isn't responding well to the question.

Finally I found a method by using this line:
`x0,y0,x1,y1,confi,cla = results.xyxy[0][1].numpy()`

**Comment by CristianR91** [comment]
Hi, 

I used this code, I found in a tutorial. https://github.com/AprendeIngenia/Parqueadero-Automatizado-con-AI
info = detect.pandas().xyxy[0]
info2 = detect.pandas().xyxy[0].to_dict(orient = "records")
    if len(info2) != 0:
        for result in info2:
            information = result['label']

**Comment by Lupi91** [comment]
Hi,
I used iterrows() like this:

pred = results.pandas().xyxy[0]
for index, row in pred.iterrows():
    #print(row['class'], row['confidence'])
    if str(row['class']) == "0":
        print("PERSON FOUND")
        if float(row['confidence']) > 0.7:
            print("HIT")

**Comment by glenn-jocher** [comment]
@Lupi91 great to hear that you found a solution! Using the `iterrows()` method is a suitable approach for iterating through the results. Your code looks good, and it's logical to access the class and confidence values for filtering the results. Keep up the good work and feel free to ask if you have any further questions or need assistance with anything else!
